tests/e2e/default/users/bulk-upload.wdio-spec.js

```diff
@@ -22,7 +22,7 @@ describe('Bulk User Creation ->', () => {
   
   it('should show an upload summary with a successful upload and a failed upload', async () => {
     const path = require('path');
-    const filePath = path.join(__dirname, 'bulk-upload-test.csv');
+    const filePath = path.join(__dirname, 'forms', 'bulk-upload-test.csv');
```

Comment:
Since `bulk-upload-test.csv` isn't a form, I think we can take it out and leave it same level as `bulk-upload.wdio-spec.js`.
What do you think?

Reply:
I think you are right, thanks!
I will move it